src/api/model/user_model

```diff
@@ -0,0 +1,6 @@
+from pydantic import BaseModel
+
+class User(BaseModel):
+    id: int
+    name: str
+    email: str
```

Comment:
The fields are okay for now, but 'Use' model should have more fields like password, d.o.b, created at, updated at.

I will work on a design document, so development for development reference.

Reply:
ok